main.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"sort"
+)
+
+type Record struct {
+	ID        uint
+	FirstName string
+	LastName  string
+	PhoneN    string
+}
+
+func removeEntry() (errr string) {
```

Comment:
nit: There is no need to return strings and use fatal errors. When you run the program for the first time, it's a completely legitimate case that the database file doesn't exist yet. Also, files can be read-only, so you just won't be able to remove entries from from. Just consider returning errors and use them as late as possible.